pkg/local_object_storage/metabase/delete.go

```diff
@@ -35,6 +36,11 @@ func (d DeleteRes) RawObjectsRemoved() uint64 {
 	return d.rawRemoved
 }
 
+// RemovedObjectSizes returns the sizes of removed objects.
+func (d DeleteRes) RemovedObjectSizes() []uint64 {
```

Comment:
i'd clarify the slice semantics in docs. IIUC it corresponds to parameterized address list. Zero values are also not so obvious